### PR TITLE
Spaceplatform (Update)

### PR DESCRIPTION
**Pull request by dvir001**
## About the PR
spaceplatform for space dungeons updated version of file to 6 (Open and resave) 

Added:
angularDamping: 999999
linearDamping: 999999

So they cannot be moved anymore
 
## Why / Balance
Too many times someone move trustees on them / push them into frontier. 

## How to test
Open admin tab
Look for all grids and inspect a grid that have a dungeon on it
Note the shuttle comp having the new numbers.

## Media
- [X] this PR does not require an ingame showcase

## Breaking changes
N/A

**Changelog**
N/A

### CODE REVIEW

File: Resources/Maps/_NF/Dungeon/spaceplatform.yml [hunk 1]
```diff
@@ -0,0 +1,2600 @@
...
+          version: 6
+    - type: Broadphase
+    - type: Physics
+      bodyStatus: InAir
```
Comment (whatston3) [inline Resources/Maps/_NF/Dungeon/spaceplatform.yml:161]: I'll need to look at what InAir actually does.

File: Resources/Maps/_NF/Dungeon/spaceplatform.yml [hunk 2]
```diff
@@ -0,0 +1,2600 @@
...
+      linearDamping: 999999
+    - type: GridPathfinding
+    - type: Gravity
+      gravityShakeSound: !type:SoundPathSpecifier
+        path: /Audio/Effects/alert.ogg
+    - type: DecalGrid
+      chunkCollection:
+        version: 2
+        nodes: []
+    - type: GridAtmosphere
```
Comment (whatston3) [inline Resources/Maps/_NF/Dungeon/spaceplatform.yml:180]: This changed from DecalGrid.  Not sure which is correct, but it seems swapped.

File: Resources/Maps/_NF/Dungeon/spaceplatform.yml [hunk 3]
```diff
@@ -0,0 +1,2600 @@
...
+      fixtures: {}
+    - type: OccluderTree
+    - type: Shuttle
+      angularDamping: 999999
+      linearDamping: 999999
+    - type: GridPathfinding
+    - type: Gravity
+      gravityShakeSound: !type:SoundPathSpecifier
+        path: /Audio/Effects/alert.ogg
+    - type: DecalGrid
```
Comment (whatston3) [inline Resources/Maps/_NF/Dungeon/spaceplatform.yml:176]: This changed from GridAtmosphere.  Not sure which is correct, but it seems swapped.

**Review by whatston3 — COMMENTED** [review]
A few questions on this one - seems like some things have changed in the file, not sure if that's expected.

**Comment by dvir001** [comment]
Just re saved it to get it to v6 after adding the angularDamping & linearDamping in game.
@whatston3 You can test it by doing the same, take the file before this PR and ``savegrid`` it again.

**Review by whatston3 — APPROVED** [review]
Seems to work.

**Comment by dvir001** [comment]
> One thing that's interesting - the Physics component here has angular and linear damping as well as the ShuttleComponent. The bluespace event grids, whenever they spawn in, do not.

Its actully added when you resave them or when they first spawn in from the shuttle comp

**Pull request merged**